nova_plugin/server.py

```diff
@@ -253,7 +253,12 @@ def rename(name):
     ctx.logger.debug(
         "server.create() server before transformations: {0}".format(server))
 
-    _handle_image_or_flavor(server, nova_client, 'image')
+    if 'block_device_mapping' in server:
+        # if there is a connected boot volume, don't require the `image`
+        # property.
+        server['image'] = ''
```

Comment:
The python-novaclient server.create() function requires an `image` input even though the api doesn't, so this is for that. 
It should probably still be improved to not overwrite the image if the user did supply one though (perhaps they know which image their volume came from and are being helpful to others looking at the console), so I'll do that & add comments